From khali@linux-fr.org Wed Dec  7 12:57:29 2005
Date: Wed, 7 Dec 2005 21:54:26 +0100
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Subject: i2c: Documentation update
Message-Id: <20051207215426.247fa402.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-driver-owner-cleanup-03.patch

Update the i2c documentation to reflect the recent change to
i2c_add_driver.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 Documentation/i2c/porting-clients |    5 ++---
 Documentation/i2c/writing-clients |    1 -
 2 files changed, 2 insertions(+), 4 deletions(-)

--- gregkh-2.6.orig/Documentation/i2c/porting-clients
+++ gregkh-2.6/Documentation/i2c/porting-clients
@@ -126,9 +126,8 @@ Technical changes:
 * [Driver] The flags field of the i2c_driver structure is gone.
   I2C_DF_NOTIFY is now the default behavior.
   The i2c_driver structure has a driver member, which is itself a
-  structure, those owner and name members should be initialized to
-  THIS_MODULE and a driver name string, respectively. i2c_driver
-  itself has no name member anymore.
+  structure, those name member should be initialized to a driver name
+  string. i2c_driver itself has no name member anymore.
 
 Coding policy:
 
--- gregkh-2.6.orig/Documentation/i2c/writing-clients
+++ gregkh-2.6/Documentation/i2c/writing-clients
@@ -26,7 +26,6 @@ address.
 
 static struct i2c_driver foo_driver = {
 	.driver = {
-		.owner	= THIS_MODULE,
 		.name	= "foo",
 	},
 	.attach_adapter	= &foo_attach_adapter,
